junit4/src/main/java/org/jboss/weld/junit4/WeldInitiator.java

```diff
@@ -108,7 +111,15 @@ public static WeldInitiator performDefaultDiscovery() {
      * @see #of(Class...)
      */
     public static Builder from(Class<?>... beanClasses) {
-        return from(createWeld().beanClasses(beanClasses));
+        Weld weld = createWeld();
+        Arrays.stream(beanClasses).forEach(clazz -> {
```

Comment:
Is there a reason to use a stream instead of a simple foreach?

Reply:
Not really, for some reason I just 'felt like' putting a stream there.
Should I turn it into plain foreach?

Reply:
If you don't mind... I think it's a good practice not to create objects (esp. streams and lambdas) unless really needed.